recipe_scrapers/coopse.py

```diff
@@ -0,0 +1,110 @@
+# mypy: allow-untyped-defs
+import json
+import re
+
+import requests
+
+from recipe_scrapers._grouping_utils import IngredientGroup
+
+from ._abstract import AbstractScraper
+
+SCRIPT_PATTERN = re.compile('"recipeId":"(?P<recipeId>[0-9]+)"')
+
+
+class CoopSE(AbstractScraper):
+    def __init__(self, url, proxies=None, timeout=None, *args, **kwargs):
+        super().__init__(url=url, proxies=proxies, timeout=timeout, *args, **kwargs)
+        scripts = self.soup.find_all("script")
+        recipe_id = None
+        for script in scripts:
+            matches = SCRIPT_PATTERN.search(str(script.string))
+            if matches:
+                data = matches.groupdict()
+                recipe_id = data["recipeId"]
+        data_url = f"https://proxy.api.coop.se/external/recipe/recipes/{recipe_id}?api-version=v1"
```

Comment:
@henrikolsson from looking at the page, I guess not, but I'll ask anyway to check: there doesn't seem to be a way to retrieve the data for a recipe on coop.se without making an additional HTTP request?

Reply:
It's a shame we need to do that initial request when all the data is in the JSON. But I haven't found another way.

Reply:
Ok, no problem - thanks for checking :+1: 

FYI: This means that the scraper may be incompatible with the work-in-progress `v15` branch, where most network requests are removed, and the library operates only on received HTML content.  There's no planned timeline for the release of v15 yet, though.